youtube.module

```diff
@@ -598,19 +598,34 @@ function youtube_field_widget_form(&$form, &$form_state, $field, $instance, $lan
       '#suffix' => '</div>',
       '#weight' => 1,
     );
-  }
 
+  }
   if (isset($start_time) && $start_time > 0) {
     $element['video_start_time'] = array(
       '#prefix' => '<div class="youtube-video-start-time">',
       '#markup' => t('YouTube video start time: !video_start_time', array(
-        '!video_start_time' => date("H:i:s", $start_time),
+        '!video_start_time' => gmdate("H:i:s", $start_time),
       )),
       '#suffix' => '</div>',
       '#weight' => 1,
     );
   }
-
+  // If Video ID, and optionally start time, are shown, show text to explain
+  // that values will be updated after the node is saved.
+  if (isset($element['video_id'])) {
+    if (isset($element['video_start_time'])) {
+      $explainer_text = t('Video ID and start time will reflect any changes in the URL after the node is saved.');
+    }
+    else {
+      $explainer_text = t('Video ID will reflect any changes in the URL after the node is saved.');
```

Comment:
The term 'node' isn't appropriate for editors. Maybe replace it with 'form' in this case?

Reply:
Good point. Thanks